### PR TITLE
Add @sarahbenmessaoud

### PR DESCRIPTION
**Pull request by sarahbenmessaoud**

### CODE REVIEW

File: sarahbenmessaoud.svg [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+<path
+  d="M 95 20 L 60 20 L 60 100 z" fill="bisque"
+/>
+  <text x="150" y="115" fill="blue" stroke="none" font-size="40" font-weight="700">ART
+    <tspan x="250" y="45" fill="red" font-size="14">What is the purpose of</tspan></text>/>
+     </g>
+</svg>
+
+
+<!-- NOTHING TO SEE HERE... MOVE ALONG!!! -->
```
Comment (wooorm) [inline sarahbenmessaoud.svg]: Three more thing!

1. Remove the below `<script>` (and its contents)
2. Add a `font-family` on any element
3. Add a `opacity` attribute on any element

File: sarahbenmessaoud.svg [hunk 2]
```diff
@@ -0,0 +1,106 @@
+<svg viewBox="0 -25 400 200" xmlns="http://www.w3.org/2000/svg">
```
Comment (wooorm) [inline sarahbenmessaoud.svg]: Could you remove this file? You can do that by going to your fork, to this file, and then clicking the trash icon in the top-right corner.

File: sarahbenmessaoud.svg (hunk 1 above)
Comment (sarahbenmessaoud) [reply sarahbenmessaoud.svg]: Oeeh wist niet dat de laatste twee verplicht waren, maar heb ze toegevoegd!

**Comment by wooorm** [comment]
Can you remove the superfluous file? Go to https://github.com/sarahbenmessaoud/course-17-18/blob/play/sarahbenmessaoud.svg and click the trash icon on the top-right.

**Comment by wooorm** [comment]
Your work is late and I unfortunately don’t have to time to review student work passed the due date.

As your work is already late you won’t receive [points for handing it in on time](https://github.com/cmda-fe3/course-17-18/blob/master/participation.md), but otherwise this won’t affect your participation grade.

**Pull request closed without merging**